Source/Blazorise/Themes/ThemeGenerator.cs

```diff
@@ -109,6 +109,23 @@ protected virtual void GenerateBodyVariables( Theme theme )
         {
             Variables[ThemeVariables.BodyTextColor] = ToHex( ParseColor( theme.BodyOptions.TextColor ) );
         }
+
+        if ( theme.BodyOptions.FontOptions is not null )
+        {
+            var fontOptions = theme.BodyOptions.FontOptions;
+
+            if ( !string.IsNullOrEmpty( fontOptions.Family ) )
```

Comment:
I generally like to do whitespace just to be a bit more "safe". Otherwise it' fine anyway.

Reply:
Kinda prefer `IsNullOrEmpty`.